thavalon-server/src/lobby/lobby_impl.rs

```diff
@@ -201,6 +285,9 @@ impl Lobby {
                 }
                 LobbyCommand::Ping { client_id } => self.send_pong(client_id).await,
                 LobbyCommand::StartGame => self.start_game().await,
```

Comment:
Will we restrict who can start the game, or is any player able to?

Reply:
Any player can for now, and probably for prealpha as well.